### ISSUE TITLE
[BITV/A11Y] Accessibility Issues

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/files_pdfviewer#630]
Moved over from https://github.com/nextcloud/server/issues/21443

* Using label elements to associate text labels with form controlsCannot control pdf viewer , it is another third party application within the application. There is a download option available to avoid using the viewer.
* Zooming and Scaling to be turned onCannot control pdf viewer , it is another third party application within the application. There is a download option available to avoid using the viewer.
* Keyboard focus flow is not easily programmed, multiple rewrites would be needed. Focus is still there but just not in the expected flow at times.
* Link at the top of each page to skip to main content areaCannot control pdf viewer , it is another third party application within the application. There is a download option available to avoid using the viewer.
* Elements should not have tabindex greater then zeroCannot control pdf viewer , it is another third party application within the application. There is a download option available to avoid using the viewer.
* No aria-expandCannot control pdf viewer , it is another third party application within the application. There is a download option available to avoid using the viewer.

**Comment by PVince81** [comment]
we should at least update pdf.js to the latest version, in case some of these are fixed

also, if they tested NC <= 22 it's likely that we already updated it recently

**Comment by PVince81** [comment]
looks like master is already on the latest version https://www.npmjs.com/package/pdfjs-dist (2.14.305) from 2 months ago

**Comment by AndyScherzinger** [comment]
The above (potentially still existing) issues would still need to be checked.
